spec/components/chart_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+data = {
+  caption: "Unique page views",
+  h_axis_title: "Date",
```

Comment:
I have noticed that in `data` there are attributes  are not tested. I would try to keep the fixture size as close as possible to the tests. I would remove the fields that are not tested, or just add a test for them if needed.